crashtracker/src/receiver.rs

```diff
@@ -64,12 +66,22 @@ pub fn receiver_entry_point_unix_socket(socket_path: impl AsRef<str>) -> anyhow:
     // Dropping the stream closes it, allowing the collector to exit if it was waiting.
 }
 
+pub fn receiver_timeout() -> Duration {
+    if let Ok(s) = std::env::var("DD_CRASHTRACKER_RECEIVER_TIMEOUT_MS") {
+        if let Ok(v) = s.parse() {
+            return Duration::from_millis(v);
+        }
+    }
+    // Default value
+    Duration::from_millis(5000)
```

Comment:
Right now this default is identical to the default we use for the collector timeout.  I think, ideally, this timeout will be shorter than the other timeout, so we have time to send a message somewhere saying what's going to happen.

(in a future PR, that other timeout will be enforced from outside of the collector, so I don't think we can necessarily assume the collector isn't responding to its own timeouts if IPC stalls)